packages/asset-swapper/src/utils/market_operation_utils/index.ts

```diff
@@ -179,9 +157,7 @@ export class MarketOperationUtils {
                 rawTwoHopQuotes,
                 isTxOriginContract,
             ],
-            offChainBalancerQuotes,
-            offChainCreamQuotes,
-        ] = await Promise.all([samplerPromise, offChainBalancerPromise, offChainCreamPromise]);
+        ] = await Promise.all([samplerPromise, cacheRefreshPromises]);
```

Comment:
is this going to refresh the caches even on a BSC API request?

Reply:
Yesss but I think that's what it did before. What's the desired behaviour?

Reply:
I don't have a strong preference, but I lean towards not refreshing the caches for a BSC request. Should be fine either way though